app/templates/single_resource/index.html

```diff
@@ -20,8 +20,8 @@ <h2 class="ui dividing header">Resources</h2>
                             <a href="{{ url_for('single_resource.edit', resource_id=resource['id']) }}">{{ resource['name'] }}</a>
                         </td>
                         <td>{{ resource['address'] }}</td>
-                        <td>{{ resource['latitude'] | round }}</td>
-                        <td>{{ resource['longitude'] | round }}</td>
+                        <td>{{ resource['latitude'] }}</td>
```

Comment:
Ah I see. I think we should leave them in and instead ensure that every resource has a lat/long. Since the front-end has no "list view" for the resources, a resource without a lat/long won't show up at all for a user.